src/Stripe.net.Tests/charges/when_creating_a_charge_with_a_stripe_account.cs

```diff
@@ -0,0 +1,43 @@
+using Machine.Specifications;
```

Comment:
I went with the `Stripe.net.Tests` because that's where all the others are for that specific deserialization use-case.

Reply:
Ack. Not ideal, but understandable :)